org.ryujinx.Ryujinx.json

```diff
@@ -0,0 +1,1291 @@
+                     "url": "https://api.nuget.org/v3-flatcontainer/runtime.win.system.net.sockets/4.3.0/runtime.win.system.net.sockets.4.3.0.nupkg",
+                     "sha512": "d70b564d53be12665bd4ddb4ce64e60fef42d17c68368a6c7877e059f41f3218bbb72867162b740bf3bd363304397eacbda8c48ea4f83d2d3ec958c11d23493d",
+                     "dest": "nuget-sources",
+                     "dest-filename": "runtime.win.system.net.sockets.4.3.0.nupkg"
+                 },
+                 {
+                     "type": "file",
+                     "url": "https://api.nuget.org/v3-flatcontainer/pangosharp/3.22.25.56/pangosharp.3.22.25.56.nupkg",
+                     "sha512": "9261f7de2c44ca57ce1b234b6a425ebac9a968128b37c244d30d0caac6a99bcf674a86471e5bfc4db7e987dbfcb259a955d23db5ad97f1e7151ac6e4a2b79eae",
+                     "dest": "nuget-sources",
+                     "dest-filename": "pangosharp.3.22.25.56.nupkg"
+                 },
+                 {
+                     "type": "file",
+                     "url": "https://api.nuget.org/v3-flatcontainer/runtime.any.system.reflection.extensions/4.3.0/runtime.any.system.reflection.extensions.4.3.0.nupkg",
+                     "sha512": "8de7a4c53fc0324e766bfec360342ee4a4b99a5975a9d61faab0a715ef71ff97aa83383a5a8affb354c02a4e2fbbb91e1b4ae6b282d2880108cb489f06aba500",
+                     "dest": "nuget-sources",
+                     "dest-filename": "runtime.any.system.reflection.extensions.4.3.0.nupkg"
+                 },
+                 {
+                     "type": "file",
+                     "url": "https://api.nuget.org/v3-flatcontainer/runtime.any.system.text.encoding.extensions/4.3.0/runtime.any.system.text.encoding.extensions.4.3.0.nupkg",
+                     "sha512": "656aa8bd9d7e19534964ac7b8405615f00359779e322d4cfe1f18c132fec4a4f52c5588bfe61cec9966a9142a73315f5d2b9e5a7c524b418364f0322b20961c3",
+                     "dest": "nuget-sources",
+                     "dest-filename": "runtime.any.system.text.encoding.extensions.4.3.0.nupkg"
+                 },
+                 {
+                     "type": "file",
+                     "url": "https://api.nuget.org/v3-flatcontainer/system.security.claims/4.3.0/system.security.claims.4.3.0.nupkg",
+                     "sha512": "ab72b90801f6c051a2b31645448eebfca74642b3cfa1d51f80e21a0d0d7ad44d3366dea139347e2852781b7f3bae820df16c3eb188a2c96244df05394ed72c86",
+                     "dest": "nuget-sources",
+                     "dest-filename": "system.security.claims.4.3.0.nupkg"
+                 }
+
+            ]
+        },
+        {
+            "name": "appdata",
+            "buildsystem": "simple",
+            "build-commands": [
+                "install -D -m0644 org.ryujinx.Ryujinx.appdata.xml /app/share/metainfo/org.ryujinx.Ryujinx.appdata.xml"
+            ],
+            "sources": [
+                {
+                    "type": "file",
+                    "path": "org.ryujinx.Ryujinx.appdata.xml"
```

Comment:
New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html

Reply:
Not sure what metainfo is

Reply:
Newer appdata files should use .metainfo.xml extension instead & installed under /metainfo instead of /appdata. See the specs

Reply:
thanks Bilel, TL;DR; hehehe